lib/fastlane/plugin/emerge/helper/github.rb

```diff
@@ -39,6 +39,17 @@ def self.base_sha
         end
       end
 
+      def self.previous_sha
+        if is_push?
+          github_event_data.dig(:before)
+        else
+          shell_command = "git rev-parse HEAD^"
+          UI.command(shell_command)
+          stdout, _, status = Open3.capture3(shell_command)
+          stdout.strip if status.success?
+        end
```

Comment:
I noticed we don't do this for other methods in this file, but I don't want it to be empty when it's still possible to retrieve. 

let me know if this is not kosher 🙏